tests/bson/bson-toPHP_error-006.phpt

```diff
@@ -18,45 +18,66 @@ $tests = [
     pack('VCa*xVCa*xxCa*xVCa*xxx', 31, 0x03, 'e1', 9, 0x06, 'u1', 0x03, 'e2', 9, 0x06, 'u2'),
 ];
 
+ini_set("mongodb.debug", "stdout");
```

Comment:
The test filename is `bson-toPHP_error-006.phpt` and the title still says "BSON decoding throws multiple exceptions". I think we should at least rename the title, but I don't feel strongly about the filename (technically, `toPHP()` is no longer erroring, just silently logging a warning).

Also, this test doesn't do much more than create an array of bytes and call `toPHP()`. Would it make sense to move `mongodb.debug` to an `--INI--` block instead of using `ini_set()`? We don't do anything with the driver by including `tools.php`, so that shouldn't produce any clutter in the logs.

Reply:
You still get lots of libmongoc initialisation messages, which we don't need. So I'm keeping that.

I've changed the titles, but not the file names.